packages/kbn-unified-data-table/src/utils/get_render_cell_value.tsx

```diff
@@ -101,10 +90,14 @@ export const getRenderCellValueFn = ({
             fieldFormats,
             closePopover,
           })}
-        </>
+        </span>
       );
     }
 
+    if (typeof row === 'undefined') {
+      return <span className={CELL_CLASS}>-</span>;
+    }
```

Comment:
Just curious why these lines were moved below since it seems like it works the same as before with the additional `&& row` check above?

Reply:
You are right. It is a strange change and I am not sure why this was done. I have reverted it and you can check it [here](https://github.com/elastic/kibana/pull/176064/files#diff-795b324c9b066e6ec4751980f32f51c0c92cf258e21161f73f2e7ba8db95be95)